### ISSUE TITLE
How to save the model after training ??

### ISSUE THREAD

**Issue by sourangshupal** [Tessellate-Imaging/monk_v1#9]
How to save the model after doing successful training in Keras or Pytorch in monk ??

**Comment by abhi-kumar** [comment]
The model is autosaved.

You will find four kinds of models
- final (the one resulting from the final epoch)
- best (the one resulting from the epoch having lowest validation loss)
- resume (the one created if training is interrupted)
- a set of models each from every epoch (this can be set as false using 
`ptf.update_save_intermediate_models(False) function`

You will find all these models in the directory
workspace
    |------<your project>
                   |---------<your experiment>
                                       |------output
                                                     |----------models
workspace/<your project>/<your experiment>/output/models

To load the model for inference invoke the experiment with eval_infer flag.
run
`ptf.Prototype("<your project>", "<your experiment>", eval_infer=True)`
and the model will be auto-loaded.

Comment if you still have any doubts on the same.
